CONTRIBUTING.md

```diff
@@ -129,6 +129,32 @@ Unit tests are black box tests that test the public interface of a class. They a
 
 Integration tests are composition of more than one class and test their interaction. They are optional for new code.
 
+## Coverage Scan
+
+To ensure that the provided testcode covers the productive code you can do a coverage scan with gcov. The reporting is done with lcov and htmlgen.
+You will need to install the following packages:
+    ```
+    sudo apt install lcov
+    ```
+
+In iceoryx we have multiple levels for testcoverage: 'unit', 'integration', 'component' and ’all’ for all levels together. You can create reports for these different testlevels or for all tests. Coverage is here done with gcc.
+The coverage scan applies to Quality level 3 - 5 and partly 4.
```

Comment:
isn't 4 between 3 and 5 ;)
Maybe you did just mean `3, 5`?
As far as I understood the quality levels, the test coverage should map only to one level. Since the higher levels inherit from the lower levels. @mossmaurice can you clarify?

Reply:
I made this statement more clear